crowd_sim/envs/crowd_sim.py

```diff
@@ -248,6 +297,13 @@ def get_human_times(self):
         del sim
         return self.human_times
 
+    def generate_robot_goal(self, goal_range = 8):
+        # self.robot_gx = (np.random.random() - 0.5) * goal_range
+        # self.robot_gy = (np.random.random() - 0.5) * goal_range
+        angle = np.random.random() *  2 * pi
+        self.robot_gx = goal_range * np.cos(angle)
+        self.robot_gy = goal_range * np.sin(angle)
+
```

Comment:
I think we can name this function more generally:
```
    def generate_agent_goal(self, goal_range = 8):
```

Then we can have it return the values, which we use later
```
        gx = goal_range * np.cos(angle)
```

Also, adding random perturbations is still a good idea. We can make it optional if we wish.

```suggestion
    def generate_agent_goal(self, goal_range = 8, perturb = False, perturb_range = 1):
        if perturb:
          px = (np.random.random() - 0.5) * perturb_range
          py = (np.random.random() - 0.5) * perturb_range
        else:
          px = 0
          py = 0
        angle = np.random.random() *  2 * pi
        gx = goal_range * np.cos(angle) + px
        gy = goal_range * np.sin(angle) + py
        return gx,gy
        
```

Later we can call the function to set the class attributes
```
self.robot_gx, self.robot_gy = self.generate_agent_goal()
````